Resources/Locales/es_es/openspades.po

```diff
@@ -0,0 +1,1538 @@
+"X-Generator: Poedit 1.8.7.1\n"
+"X-Poedit-SourceCharset: UTF-8\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+#: Sources/Client/Client.cpp:582
+msgctxt "Client"
+msgid "Map saved: {0}"
+msgstr "Mapa guardado: {0}"
+
+#: Sources/Client/Client.cpp:586 Sources/Client/Client.cpp:592
+msgctxt "Client"
+msgid "Saving map failed: "
+msgstr "Error en el guardado del mapa: {0}"
+
+#: Sources/Client/Client.cpp:624
+msgctxt "Client"
+msgid "[Global] "
+msgstr "[Global] "
+
+#: Sources/Client/Client_Draw.cpp:119
+msgctxt "Client"
+msgid "Sceneshot saved: {0}"
+msgstr "Sceneshot guardada: {0}"
+
+#: Sources/Client/Client_Draw.cpp:121
+msgctxt "Client"
+msgid "Screenshot saved: {0}"
+msgstr "Screenshot guardada: {0}"
+
+#: Sources/Client/Client_Draw.cpp:125 Sources/Client/Client_Draw.cpp:131
+msgctxt "Client"
+msgid "Screenshot failed: "
+msgstr "Screenshot fallida: {0}"
+
+#: Sources/Client/Client_Draw.cpp:198
+msgctxt "Client"
+msgid "NOW LOADING"
+msgstr "CARGANDO"
+
+#: Sources/Client/Client_Draw.cpp:220
+msgctxt "Client"
+msgid "Disconnecting..."
+msgstr "Desconectando..."
+
+#: Sources/Client/Client_Draw.cpp:523
+msgctxt "Client"
+msgid "Out of Block"
+msgstr "Sin bloques"
+
+#: Sources/Client/Client_Draw.cpp:528
+msgctxt "Client"
+msgid "Out of Grenade"
+msgstr "Sin granadas"
+
+#: Sources/Client/Client_Draw.cpp:534
+msgctxt "Client"
+msgid "Reloading"
+msgstr "Recargando"
+
+#: Sources/Client/Client_Draw.cpp:536 Sources/Client/Client_Input.cpp:440
+msgctxt "Client"
+msgid "Out of Ammo"
+msgstr "Sin munición"
+
+#: Sources/Client/Client_Draw.cpp:539
+msgctxt "Client"
+msgid "Press [{0}] to Reload"
+msgstr "Presiona [{0}] para recargar"
+
+#: Sources/Client/Client_Draw.cpp:584
+msgctxt "Client"
+msgid "You will respawn in: {0}"
+msgstr "Reaparecerás en: {0}"
+
+#: Sources/Client/Client_Draw.cpp:586
+msgctxt "Client"
+msgid "Waiting for respawn"
+msgstr "Esperando para reaparecer"
+
+#: Sources/Client/Client_Draw.cpp:753
+msgctxt "Client"
+msgid "Following {0}"
+msgstr "Siguiendo {0}"
+
+#: Sources/Client/Client_Input.cpp:428
+msgctxt "Client"
+msgid "Out of Blocks"
+msgstr "Sin bloques"
+
+#: Sources/Client/Client_Input.cpp:452
+msgctxt "Client"
+msgid "Out of Grenades"
+msgstr "Sin granadas"
+
+#: Sources/Client/Client_NetHandler.cpp:89
+msgctxt "Client"
+msgid "{0} captured {1}'s territory"
+msgstr "{0} ha capturada el territorio de {1}"
+
+#: Sources/Client/Client_NetHandler.cpp:91
+msgctxt "Client"
+msgid "{0} captured an neutral territory"
+msgstr "{0} ha capturado un territorio neutral"
+
+#: Sources/Client/Client_NetHandler.cpp:99
+msgctxt "Client"
+msgid "{0} captured {1}'s Territory"
+msgstr "{0} ha capturada el Territorio de {1}"
+
+#: Sources/Client/Client_NetHandler.cpp:101
+msgctxt "Client"
+msgid "{0} captured an Neutral Territory"
+msgstr "{0} ha capturado un Territorio Neutral"
+
+#: Sources/Client/Client_NetHandler.cpp:127
+msgctxt "Client"
+msgid "{0} captured {1}'s intel"
+msgstr "{0} capturó la intel {1}"
```

Comment:
I only know very basic spanish, but shouldn't the translation be "{0} capturó la intel de {1}"?

Then again, there is a high chance that I might be wrong, because I don't know much spanish.